Source/CarthageKit/Project.swift

```diff
@@ -522,7 +522,7 @@ public final class Project {
 			})
 	}
 	
-	public func buildOrderForResolvedCartfile(cartfile: ResolvedCartfile) -> SignalProducer<Dependency<PinnedVersion>, CarthageError> {
+	public func buildOrderForResolvedCartfile(cartfile: ResolvedCartfile, dependenciesToInclude: [String]? = nil) -> SignalProducer<Dependency<PinnedVersion>, CarthageError> {
```

Comment:
I think it might make more logical sense to let the caller do the filtering instead.

If we feel that duplication is an issue, we could add a custom RAC operator that filters values based on a collection of values.